clients/omnitruck/platform_services.go

```diff
@@ -0,0 +1,111 @@
+package omnitruck
+
+import (
+	"github.com/chef/omnitruck-service/constants"
+	"github.com/chef/omnitruck-service/logger"
+	"github.com/gofiber/fiber/v2"
+)
+
+type PlatformServices struct {
+	Logger logger.Logger
+}
+
+func NewPlatformServices(logger logger.Logger) PlatformServices {
+	return PlatformServices{
+		Logger: logger,
+	}
+}
+
+func (r *PlatformServices) PlatformVersionsAll(req *RequestParams, serverMode int) ([]ProductVersion, error) {
+	productVersions := []ProductVersion{}
+	flags := RequestParamsFlags{
+		Channel: true,
+	}
+	requestParams := ValidateRequest(req, flags)
+	if !requestParams.Ok {
+		r.Logger.Error("", requestParams.Message)
+		return productVersions, fiber.NewError(requestParams.Code, requestParams.Message)
+	}
+	if serverMode == 2 && req.Product == constants.PLATFORM_SERVICE {
+		productVersions = append(productVersions, "latest")
```

Comment:
can we move the product version `latest` into a constant and use it where ever required? Eg: in this func- `PlatformVersionLatest`

Reply:
Done